### PR TITLE
Update qcad from 3.26.2 to 3.26.4

### PR DESCRIPTION
**Pull request by ralphrmartin**
Created with `brew bump-cask-pr`.

### CODE REVIEW

File: Casks/qcad.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 cask "qcad" do
-  version "3.26.2"
+  version "3.26.4"
 
   if MacOS.version <= :high_sierra
     sha256 "d1c06bf0f2fc7df0bbe2f428e159c7aa12f13537141bc69d8e5dee1c793047b6"
```
Comment (cho-m) [inline Casks/qcad.rb:5]: This probably needs to be updated manually.

Comment (ralphrmartin) [reply Casks/qcad.rb:5]: While there is not a new version for High Sierra, there seems to be a separate (non Rosetta 2) build for M1 macs. I'm not sure how to proceed.

Comment (cho-m) [reply Casks/qcad.rb:5]: The DMGs I see are:
- https://www.qcad.org/archives/qcad/qcad-3.26.4-trial-macos-10.10-10.13.dmg
- https://www.qcad.org/archives/qcad/qcad-3.26.4-trial-macos-10.14-11.2.dmg
- https://www.qcad.org/archives/qcad/qcad-3.26.4-trial-macos-11.0-11.2-arm64.dmg

The first matches this branch, so SHA should be updated or `brew install` will fail on SHA mismatch error.

For ARM support, you will want to modify `else ... end` section to:
```
elsif Hardware::CPU.intel?
  sha256 ...
  url ...
else
  sha256 ...
  url ...
end
```

Comment (ralphrmartin) [reply Casks/qcad.rb]: I have now committed the needed change.

Comment (ralphrmartin) [reply Casks/qcad.rb]: Fixed indentation

Comment (ralphrmartin) [reply Casks/qcad.rb]: Trailing whitespace causes an error? What a rotten language design decision.

**Comment by ralphrmartin** [comment]
I dont get it. It says the whitespace problem is "autocorrectable", yet does not do so. Why?

**Review by miccal — APPROVED** [review]

**Pull request merged**